index.js

```diff
@@ -216,6 +224,28 @@ function getPostDate(post) {
 	return luxon.DateTime.fromRFC2822(post.pubDate[0], { zone: 'utc' }).toISODate();
 }
 
+function getTags(post) {
+	if (typeof post.category === 'undefined' || post.category === null) {
+		return '[]';
+	}
+	let tagDomains = argv.categoriestotags
+		? ['post_tag']
+		: ['post_tag', 'category'];
+	let tags = post.category.filter(cat => tagDomains.includes(cat.$.domain)).map(tag => tag.$.nicename);
+	return '[' + tags.join(", ") + ']';
```

Comment:
The format you're using here returns something like this:

```markdown
tags: [css, mobile, responsive-web-design]
```

But AFAIK the markdown frontmatter format for arrays should be something like this:

```markdown
tags:
    - css
    - mobile
    - responsive-web-design
```

This seems to be the case for Jekyll and Gatsby at least. Curious if you've seen otherwise.

Reply:
Both are valid, the latter takes too much space for my taste :) https://yaml.org/spec/1.2/spec.html#id2759963